modules/oadp-creating-restore-cr.adoc

```diff
@@ -55,3 +55,15 @@ $ oc get restore -n openshift-adp <restore> -o jsonpath='{.status.phase}'
 $ oc get all -n <namespace> <1>
 ----
 <1> Namespace that you backed up.
+
+. If you use Restic to restore `DeploymentConfig` objects or if you use post-restore hooks, run a cleanup script by entering the following command:
++
+[source,terminal]
+----
+$ bash dc-restic-post-restore.sh <restore-name>
+----
+
+[NOTE]
+====
+In the course of the restore process, the OADP Velero plug-ins scale down the `DeploymentConfig` objects and restore the pods as standalone pods to prevent the cluster from deleting the restored `DeploymentConfig` pods immediately on restore and to allow Restic and post-restore hooks to complete their actions on the restored pods. The cleanup script removes these disconnected pods and scale any `DeploymentConfig` objects back up to the appropriate number of replicas.
```

Comment:
Is this the only script that users are supposed to use for cleanup? Is this the only time they are likely to need to run it? If so, consider putting the script in-line in a [collapsible block](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#collapsible-content) and reworking this note into a step (either its own, or a previous step).

If not, you need to add an introductory statement to the script and consider moving it closer to this procedure in the assembly structure.

Reply:
@kalexand-rh 
Yes, this is the only script that users are supposed to use for cleanup and the only time to run it. Running the script is Step 4, and the note was intended to be an explanation for this step and should've been indented (corrected now). Thank you for suggesting a collapsible block for the script! Please take a look now.